### PR TITLE
AG-9346 Exporting data from coredb to pynamodb

### PR DESCRIPTION
**Pull request by jose-aiq**
## Description of the change

> Exporting data from coredb to pynamodb

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Task [#AG-9346](https://agentiq.atlassian.net/browse/AG-9346) 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: dags/api_exports/run_exports.py [hunk 1]
```diff
@@ -139,17 +152,42 @@ def validate_conversations(validation_fields, start_date, end_date, coredb_id_fi
         is_valid = conversations_validator.validate(conversations, coredb_id_field)
         logging.info(f'Finished validating  conversations from dynamo {is_valid}')
         return is_valid
+
+    def validate_customer_feedback(validation_fields, start_date, end_date, coredb_id_field='id'):
+        logging.info(f'Validating customer feedback csat from dynamo from {start_date} to {end_date}')
+        customer_feedback_csat_validator = dynamoRecordsValidator('customer_feedback_csat',
```
Comment (slaveniq) [inline dags/api_exports/run_exports.py:158]: Is it always csat? What happens for customers who use NPS?

Comment (jose-aiq) [reply dags/api_exports/run_exports.py:158]: This method is called twice, one per each type, bellow this code is the second call to the method.

Comment (slaveniq) [reply dags/api_exports/run_exports.py:158]: To be more pythonesque and elegant it is probably better to put the validation in loop:

valid = True
types = {'nps':customer_feedback_nps_validator, 'csat':customer_feedback_csat_validator}
for t in types:
  if not types[t].validate(blah, blah, t):
    valid = False
    break
 return valid

Comment (jose-aiq) [reply dags/api_exports/run_exports.py]: Done

**Review by slaveniq — APPROVED** [review]

**Pull request merged**